tests/export/test_nemo_export.py

```diff
@@ -25,7 +25,7 @@
 try:
```

Comment:
Can we add vllm tests to this script or in a separate script?

Reply:
Done.